src/main/java/org/reactome/release/qa/check/CoV2InfectionPathwayEventCheck.java

```diff
@@ -0,0 +1,188 @@
+ * hasn't been updated. These are complementary cases, and just served to notify the curator of a possible mistake with
+ * CoV-2 instances.
+ *
+ * @author jcook
+ */
+
+@SliceQACheck
+public class CoV2InfectionPathwayEventCheck extends AbstractQACheck {
+
+    @Override
+    public QAReport executeQACheck() throws Exception {
+        QAReport report = new QAReport();
+
+        GKInstance cov2InfectionPathwayInst = dba.fetchInstance(QACheckerHelper.COV_2_INFECTION_PATHWAY_DB_ID);
+
+        // Get all Events contained within 'SARS-CoV-2 Infection' pathway.
+        for (GKInstance cov2Event : InstanceUtilities.getContainedEvents(cov2InfectionPathwayInst)) {
+            List<String> issues = new ArrayList<>();
+            // If does not have any modified instances, or if created instance predates most recent modified instance.
+            if (!hasRecentlyModifiedSummation(cov2Event)) {
+                issues.add("Summation instance has not been recently modified");
+            }
+
+            // inferredFrom being null is a proxy for curator being 'done' with instance. Therefore, if it is null and the
+            // issue is being reported, it indicates they likely made an error.
+            // If literatureReference attribute does not contain at least 1 litRef from 2020.
+            if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null && !hasRecentLiteratureReference(cov2Event)) {
+                issues.add("Does not contain a 2020 literature reference");
+            }
+            // Summation text was updated near end of release cycle where CoV-1-to-CoV-2 projections happened.
+            // If it didn't have the updated text, there are two reasons for it, outlined below.
+            if (!hasCorrectSummationFormat(cov2Event)) {
+                if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null) {
+                    issues.add("InferredFrom is empty but instance has COVID inference summation text");
+                } else {
+                    issues.add("InferredFrom is populated but instance does NOT have COVID inference summation text.");
+                }
+            }
+            if (issues.size() > 0) {
+                report.addLine(getReportLine(cov2Event, String.join("|",issues)));
+            }
+        }
+        report.setColumnHeaders(getColumnHeaders());
+        return report;
+    }
+
+    /**
+     * Checks if instance created dateTime is before most recent modified instance's dateTime. Also checks if
+     * any modified instance is present -- it should be given that there was at least 1 required modification from the
+     * CoV-1-to-CoV-2 merge.
+     * @param cov2Event - GKInstance, Event contained within 'SARS-CoV-2 Infection' Pathway.
+     * @return - boolean, true if modification property looks correct, false if not.
+     * @throws Exception, thrown by MySQLAdaptor.
```

Comment:
Similar to https://github.com/reactome/release-qa/pull/26/files#r544613631.  Just mentioning the exception is thrown if unable to get a summations or their attribute values.